codalab/apps/web/views.py

```diff
@@ -536,11 +535,20 @@ def get_context_data(self, **kwargs):
                         'counts': [s['count'] for s in qs],
                         'sorting': score_def.sorting,
                     }
-
-                context['top_three_leaders'] = self.get_object().get_top_three()
+                # Below is where we refactored top_three context.
+                context['top_three'] = context['active_phase'].scores()
```

Comment:
we can keep this behavior in `get_top_three()`? did you remove `get_top_three()` if we're not using it any more?

Reply:
I left `get_top_three()` for now.

Reply:
hmm, think we should pull it, though?

Reply:
I removed that method.